packages/rancher-logging/rancher-logging.patch

```diff
@@ -96,21 +113,33 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/values
 +  fluentbit:
 +    repository: rancher/fluent-fluent-bit
 +    tag: 1.5.4
++    nodeSelector:
++      kubernetes.io/os: linux
++    tolerations:
++      - key: node-role.kubernetes.io/controlplane
++        value: "true"
++        effect: NoSchedule
++      - key: node-role.kubernetes.io/etcd
++        value: "true"
++        effect: NoExecute
++      - key: cattle.io/os
++        operator: "Equal"
++        value: "linux"
++        effect: NoSchedule
 +  fluentd:
 +    repository: rancher/banzaicloud-fluentd
 +    tag: v1.11.2-alpine-2
++    nodeSelector:
++      kubernetes.io/os: linux
++    tolerations:
++      - key: cattle.io/os
++        operator: "Equal"
++        value: "linux"
++        effect: NoSchedule
 +  syslog_forwarder:
 +    repository: rancher/fluent-bit-out-syslog
 +    tag: 0.1.0
```

Comment:
Putting it under images feels weird. The standard would be to put things above in the `tolerations` node at the main level, but since we are splitting across fluentd/fluent-bit, we could at least make a node next to that one that splits them. Like: 
```

tolerations:
   - key: cattle.io/os

fluentbit_tolerations:
    ....
fluentD_tolerations:
    ....
```
What do you think @paynejacob

Reply:
We should probably also keep those standard `tolerations` ones working across the board, looks like they would be ignored in most places right now. It should use both.

Reply:
That makes sense. I'll adjust the PR.